pkg/cli/prompt/validator.go

```diff
@@ -77,6 +78,16 @@ func ValidateResourceName(input string) error {
 	return errors.New(invalidResourceNameMessage)
 }
 
+// ValidateApplicationName checks if the given string is a valid Application name, and returns an error if it is not.
+// The rules for application name matches that of kubernetes namespace since we use the name to also create a namespace for the application.
+func ValidateApplicationName(input string) error {
+	err := ValidateKubernetesNamespace(input)
```

Comment:
Not directly related to your PR but ValidateKubernetesNamespace seems to return invalidResourceNameMessage and invalidNamespaceNameMessage is unused. Could you fix that?